EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -66,19 +71,22 @@
      * @param page the page the panel belongs to
      */
     GeneratePanel(String id, final GeneratorPage page) {
+
         super(id);
         this.page = page;
         this.panel = this;
 
+
         layoutForm = new Form("layoutForm") {
 
             @Override
             public void onSubmit() {
                 EventGenerator eventGenerator = new EventGenerator();
-                if (eventTimestamps == null) {
-                    eventGenerator.generateEvents(eventCount, scaleFactor, selectedEventType, attributeInput);
-                } else {
-                    eventGenerator.generateEvents(eventCount, scaleFactor, selectedEventType, attributeInput, eventTimestamps);
+                if(eventTimestamps == null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') 'if' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_Around_BP2016)